runtime/interpreter/value.go

```diff
@@ -2492,21 +2500,55 @@ type IntValue struct {
 	BigInt *big.Int
 }
 
-func NewIntValueFromInt64(value int64) IntValue {
-	return NewIntValueFromBigInt(big.NewInt(value))
+func NewIntValueFromInt64(memoryGauge common.MemoryGauge, value int64) IntValue {
+	const int64Size = int(unsafe.Sizeof(int64(0)))
+	return NewIntValueFromBigInt(
+		memoryGauge,
+		common.NewBigIntMemoryUsage(int64Size),
```

Comment:
`common.NewBigIntMemoryUsage(int64Size)` is a fixed value for all Int64 values it seems. Can move to a module-level variable?

Reply:
Good point, refactored in 05a3f20542f1f94c0a04a33b7107229f7829688c